src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -47,18 +47,23 @@ type Props = {
 export default function GamesSettings({ useDetails = true }: Props) {
   const { t } = useTranslation()
   const { platform } = useContext(ContextProvider)
-  const { isDefault, gameInfo, isMacNative, isLinuxNative } =
-    useContext(SettingsContext)
+  const { isDefault, gameInfo, isLinuxNative } = useContext(SettingsContext)
```

Comment:
Can't we remove `isLinuxNative` ?

Reply:
`isLinuxNative` is required for fixing the cloud save sync displaying for GOG Linux games. That fix was pushed to the Small Fixes PR. It displayed an error whenever I removed it.

Reply:
It is needed that to fix the GOG Linux games cloud saves bug. Otherwise, it will show that setting even with using `window.api.isNative`

I pushed this fix to the now merged Small Fixes PR.